sa/storage-authority.go

```diff
@@ -429,6 +466,33 @@ func (ssa *SQLStorageAuthority) UpdatePendingAuthorization(authz core.Authorizat
 		return
 	}
 
+	var challs []challModel
+	_, err = tx.Select(
+		&challs,
+		"SELECT * FROM challenges WHERE authorizationID = :authID",
```

Comment:
Actually, this is another example where you'd be better off just calling GetAuthorization so you don't have to duplicate the code that reads in the challenges. Then you can limit this code to just the parts that save out updates.